### PR TITLE
absolute position challenge done

### PR DESCRIPTION
**Pull request by chris-to-fer**
Session: CSS positioning
Challenge: Absolute position
weather: sunny

### CODE REVIEW

File: css/css-positioning_absolute-no-advanced-requirements/css/styles.css [hunk 1]
```diff
@@ -37,28 +37,49 @@ main {
...
 .box--top-left {
   background-color: #aed6f1;
+  position: absolute;
+  top: 0;
+  left: 0;
 }
 
 .box--top-center {
   background-color: #85c1e9;
+  position: absolute;
```
Comment (j-schilling) [inline css/css-positioning_absolute-no-advanced-requirements/css/styles.css:47]: If you put position: absolute in the general .box class above, you don't have to put it in every single .box class as you did.

File: css/css-positioning_absolute-no-advanced-requirements/css/styles.css [hunk 2]
```diff
@@ -37,28 +37,49 @@ main {
...
+  top: 0;
+  right: 0;
 }
 
 .box--center {
   background-color: #3498db;
+  position: absolute;
+  top: 260px;
+  left: 210px;
 }
```
Comment (j-schilling) [inline css/css-positioning_absolute-no-advanced-requirements/css/styles.css:64]: You did it the calculating way. As we saw in the recap the better way would be to use transform: translate...

**Review by j-schilling — APPROVED** [review]
Hi Christoper, I approved and left two minor comments. :)

File: css/css-positioning_absolute-no-advanced-requirements/css/styles.css (hunk 2 above)
Comment (chris-to-fer) [reply css/css-positioning_absolute-no-advanced-requirements/css/styles.css:64]: jap

**Pull request merged**